README.md

```diff
@@ -122,7 +122,7 @@ by [ACM][acm]'s heartbeat mechanism, a failover will be triggered to all related
 
 ## Further Reading
 
-- [Deploy an example DR Application][example-app]
+- [Deploy an example DR Application](./rdr-example-app/README.md)
```

Comment:
Revert this to `[example-app]` and scroll down to the end of the README; there, you will find the link that needs to be modified.

![image](https://github.com/RHEcosystemAppEng/regional-resiliency-pattern/assets/28388442/a36c8e41-e931-49e8-abea-5665e81dec92)